scripts/install.sh

```diff
@@ -1,49 +1,94 @@
 #!/bin/bash
-set -e
+set -euo pipefail
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
 
 # Dafault inputs.
-KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
 KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
 CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
+NETWORK_DEFAULT="local"
+KEEP_ACCOUNT_PASSWORD=${KEEP_HOST_CHAIN_ACCOUNT_PASSWORD:-"password"}
+ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}
 
-# Read user inputs.
-read -p "Enter ethereum accounts password [$KEEP_ETHEREUM_PASSWORD_DEFAULT]: " ethereum_password
-KEEP_ETHEREUM_PASSWORD=${ethereum_password:-$KEEP_ETHEREUM_PASSWORD_DEFAULT}
+help()
+{
+   echo -e "\nUsage: ENV_VAR(S) $0"\
+           "--config-dir <path>"\
+           "--network <network>"
+   echo -e "\nEnvironment variables:\n"
+   echo -e "\tKEEP_HOST_CHAIN_ACCOUNT_PASSWORD: Unlock an account with a password. Default password is 'password'"
+   echo -e "\tCONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Ethereum"
+   echo -e "\nCommand line arguments:\n"
+   echo -e "\t--config-dir: Configuration path for keep-core client"
```

Comment:
Well, yes, but if we are entering `start.sh` territory, then not only $CONFIG_DIR_PATH_DEFAULT is qualified for change, but also other code.  We use "Ethereum" in many vars there, which in "Celo" world have their own. I guess what I'm saying is that we should probably change `start.sh` & add `start-celo.sh` as well. The question is if this should be a part of this PR?

Reply:
I'd change only how `CONFIG_DIR_PATH` is evaluated in `start.sh` to read it from parameters. I am fine with leaving `start.sh` as-is for now, and maybe just adding FIXME to the code saying that `install.sh` reads this value from parameters and we should make the approach consistent between these scripts.

We cannot change `KEEP_ETHEREUM_PASSWORD` in `start.sh` now because it's used in the Go code. This is out of the scope of this work.

Reply:
a152f5a